tmt/steps/__init__.py

```diff
@@ -409,12 +497,13 @@ def show(self, keys=None):
             return
         # Step name (and optional summary)
         echo(tmt.utils.format(
-            self.step, self.get('summary', ''),
+            self.step.name, self.get('summary', ''),
             key_color='blue', value_color='blue'))
         # Show all or requested step attributes
         base_keys = ['name', 'how']
         if keys is None:
             keys = self._common_keys + self._keys
+        assert keys is not None
```

Comment:
mypy should be able to recognize `keys` is no longer `None`, as long as it can inspect `self._common_keys` and `self._keys`. Is `assert` here needed/

Reply:
Hmm, I tried to remove that, and looks like it was not needed. Maybe it was resolved somewhere after I added the line.